grammar.js

```diff
@@ -283,6 +299,7 @@ module.exports = grammar({
     ),
 
     _declarator: $ => choice(
+      $.attributed_declarator,
```

Comment:
I'm not sure if attributes are allowed on struct field declarations or typedefs, but if so, I think we'll need to add something like `attributed_declarator` inside of `_field_declarator` and/or `_type_declarator`.

Are things like these valid?

```c
struct S {
  int a [[some_field_attribute]];
};

typedef int MyInt [[some_type_attribute]];
```

All the `_declarator` rules are a bit confusing; there's some duplication because the grammar tries to distinguish between `field_identifier`, `type_identifier` and `identifier`.

Reply:
Done.